src/app/models/session.rb

```diff
@@ -19,7 +19,7 @@ class Session < ActiveRecord::Base
 
   scope :with_attendence_count, -> { select('*').joins("LEFT OUTER JOIN (SELECT session_id, count(id) AS attendence_count FROM attendances GROUP BY session_id) AS attendence_aggregation ON attendence_aggregation.session_id = sessions.id") }
 
-  scope :for_current_event, -> { where(event_id: Event.current_event.id) }
+  scope :for_current_event, -> { where(event_id: Event.current_event.id) unless Event.current_event.nil? }
```

Comment:
I think we should just remove the scope entirely and grab sessions like `Event.current_event.sessions`

Reply:
That might be good in a future PR, Casey. Regardless, `for_current_event` should always return a collection, never `nil`.

No pages in the app need to work when there are no events; that’s a non-goal. The app will never be (and should not be) available to the public in that state. The fix on this line therefore is not necessary.

The test and the typo fix are good, so I'll revert this line, fix the spec to create an empty event, and merge the result.